Z/Data/Parser/Base.hs

```diff
@@ -666,12 +666,30 @@ skipWhile p =
                     in if V.null rest then Partial (go s) else k s () rest
         in go s0
 
+-- | Similar to 'skipWhile', but requires the predicate to succeed on at least one byte
+-- of input: it will fail if the predicate never returns 'True' or reach the end of input
+--
+skipWhile1 :: (Word8 -> Bool) -> Parser ()
+skipWhile1 p = do
+    bs <- takeWhile p
```

Comment:
This function should be implemented based on `skipWhile` because it needs not to accumulate results, especially when across the input chunk boundary.